dataclients/kubernetes/kube.go

```diff
@@ -348,9 +349,19 @@ func shuntRoute(r *eskip.Route) {
 }
 
 func healthcheckRoute(healthy, reverseSourcePredicate bool) *eskip.Route {
-	status := http.StatusOK
+	logFilters := []*eskip.Filter{{
```

Comment:
i think we can also call `logFilters` as just `filters`, because their main gig is not the logging part but the status.

Reply:
merged